packages/app/src/components/Admin/Notification/ManageGlobalNotification.jsx

```diff
@@ -18,313 +21,265 @@ import TriggerEventCheckBox from './TriggerEventCheckBox';
 
 const logger = loggerFactory('growi:manageGlobalNotification');
 
-class ManageGlobalNotification extends React.Component {
+const ManageGlobalNotification = (props) => {
 
-  constructor() {
-    super();
+  const [globalNotificationId, setGlobalNotificationId] = useState(null);
+  const [triggerPath, setTriggerPath] = useState('');
+  const [notifyToType, setNotifyToType] = useState('mail');
+  const [emailToSend, setEmailToSend] = useState('');
+  const [slackChannelToSend, setSlackChannelToSend] = useState('');
+  const [triggerEvents, setTriggerEvents] = useState(new Set());
```

Comment:
元のコードのように

```suggestion
  const [triggerEvents, setTriggerEvents] = useState(new Set(globalNotification.triggerEvents));
```

としてください。

また、↓の感じで書いておいていただけると。
```
let globalNotification;

//TODO: xxxxxxx タスクURL ~
// globalNotification = JSON.parse(document.getElementById('admin-global-notification-setting').getAttribute('data-global-notification'));
 ```

Reply:
上記の通り修正済み。
globalNotificationだけinitialize the valueしなければならないので、const globalNotification = null;にしてあります。